client/src/App.js

```diff
@@ -7,7 +7,7 @@ import RecommendedCategories from './Clusters/RecommendedCategories/RecommendedC
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = { text: '' };
+    this.state = { text: ""gh };
```

Comment:
? Typo?

Reply:
I was doing a manual regression test just to make sure the updated lint workflow still pucks up these errors. Removed now.